code/modules/gunse/modular_gun_parts.dm

```diff
@@ -710,7 +846,46 @@ ABSTRACT_TYPE(/obj/item/gun_parts/accessory)
 	icon_state = "juicer_drum"
 	overlay_y = 8
 
-	four
+/obj/item/gun_parts/magazine/juicer/four
 		name = "HOTTTT SHOTTS MAG"
 		max_ammo_capacity = 4
 
+/obj/item/gun_parts/magazine/soviet/ak/thirdak //no name additions for the next two. Don't change it!- eagle
+	name = "the bottom third of a AK magazine"
+	desc = "Some fuckin asshole took the other two thirds!"
+	max_ammo_capacity = 3
+	jam_frequency_reload = 1
```

Comment:
i'd still recommend about +2 or +3 jam frequency on each of these probably

Reply:
I've been toying around with a idea, see later post